datalad/cmd.py

```diff
@@ -254,7 +254,6 @@ def get_git_environ_adjusted(env=None):
         if 'GIT_SSH_COMMAND' not in git_env:
             git_env['GIT_SSH_COMMAND'] = GIT_SSH_COMMAND
             git_env['GIT_SSH_VARIANT'] = 'ssh'
-        git_env['GIT_ANNEX_USE_GIT_SSH'] = '1'
```

Comment:
Why is it "rogue" and removed?
Seems was added for a reason: https://github.com/datalad/datalad/commit/6320c3f826a1335c301b57889a472685f21b82d7

Reply:
Definitely OOS for the purpose/title of the PR

Reply:
Yepp. I put it back. 
Thx